### PR TITLE
flake.lock: update

### PR DESCRIPTION
**Pull request by andresilva**
#### Describe your PR, what does it fix/add?

We need hyprwayland-scanner 0.3.5.

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)

No

#### Is it ready for merging, or does it need work?

Yes

### CODE REVIEW

File: nix/default.nix [hunk 1]
```diff
@@ -5,7 +5,6 @@
   pkgconf,
   makeWrapper,
   cmake,
-  meson,
```
Comment (vaxerski) [inline nix/default.nix:8]: what is this troll, I don't think this is a flake bump

**Comment by vaxerski** [comment]
took just the hw-s bump and committed to main

File: nix/default.nix (hunk 1 above)
Comment (andresilva) [reply nix/default.nix:8]: It's unused 😶‍🌫️

Comment (fufexan) [reply nix/default.nix:8]: oh, forgot about it.

**Pull request closed without merging**